notes-service/src/main/java/org/exoplatform/wiki/service/impl/NoteServiceImpl.java

```diff
@@ -242,7 +242,7 @@ public Page createNote(Wiki noteBook, String parentNoteName, Page note, Identity
 
       // call listeners
       postAddPage(noteBook.getType(), noteBook.getOwner(), note.getName(), createdPage);
-
+      Utils.broadcast(listenerService, "note.posted", note.getLastUpdater(), createdPage);
```

Comment:
We should broadcast listeners as well in `public Page createNote(Wiki noteBook, Page parentPage, Page note)` to ensure information consistency & coherence even in batch method.